src/libcrun/status.c

```diff
@@ -137,14 +137,14 @@ libcrun_write_container_status (const char *state_root, const char *id, libcrun_
 
   ret = read_pid_stat (status->pid, &st, err);
   if (UNLIKELY (ret))
-    return crun_make_error (err, errno, "parse state file");
+    return crun_make_error (err, 0, "parse state file");
```

Comment:
here we should just do a `return ret;` since `err` is generated in `read_pid_stat`

Reply:
I thought that multi-level wrapping is OK (did not take a deep loop I admit).

fixed